resources/views/Widgets/OrderConfirmation/OrderDocumentsWidget.twig

```diff
@@ -0,0 +1,80 @@
+
+{{ Twig.set("enabledRoutes", Twig.call("config", ["IO.routing.enabled_routes"]) ~ " | split(\", \")") }}
+{{ Twig.set("translationMap",
+    {
+        invoice:            "Ceres::Template.orderConfirmationOrderDocumentsInvoice",
+        invoice_external:   "Ceres::Template.orderConfirmationOrderDocumentsInvoiceExternal",
+        delivery_note:      "Ceres::Template.orderConfirmationOrderDocumentsDeliveryNote",
+        order_confirmation: "Ceres::Template.orderConfirmationOrderDocumentsOrderConfirmation",
+        pickup_delivery:    "Ceres::Template.orderConfirmationOrderDocumentsPickupDelivery",
+        reversal_document:  "Ceres::Template.orderConfirmationOrderDocumentsReversalDocument"
+    } | json_encode)
+}}
+
+{{ Twig.set("orderDocuments", "[]") }}
+{{ Twig.for("document", "data.order.documents | default([])", "translationMap[document.type] is defined") }}
+    {{ Twig.set("orderDocuments", "orderDocuments | merge([document])") }}
+{{ Twig.endfor() }}
+
+{{ Twig.if("('order-document' in enabledRoutes or 'all' in enabledRoutes) and (orderDocuments | length > 0 or #{ isPreview | json_encode })") }}
+    <div class="widget widget-order-documents widget-{{ appearance }}{% if customClass | length > 0 %} {{ customClass }}{% endif %}{% if spacingMargin | length > 0 %} {{ spacingMargin }}{% endif %}"
+            {% if inlineMargin | length > 0 %} style="{{ inlineMargin }}"{% endif %}>
+
+        {{ Twig.if("orderDocuments | length > 1 and ('order-document' in enabledRoutes or 'all' in enabledRoutes) or #{ isPreview | json_encode }") }}
+        {# dropdown for multiple documents #}
+            <div class="btn-group">
+                <button type="button"
+                    class="btn btn-appearance dropdown-toggle {{ buttonSize }}{% if spacingPadding | length > 0 %} {{ spacingPadding }}{% endif %}"
+                    {% if inlinePadding | length > 0 %} style="{{ inlinePadding }}"{% endif %}
+                    {{ Twig.if("orderDocuments | length > 0") }}
+                        data-toggle="dropdown"
+                        aria-haspopup="true"
+                        aria-expanded="false"
+                    {{ Twig.endif() }}>
+                    {{ Twig.trans("Ceres::Template.orderConfirmationOrderDocuments") }}
+                </button>
+
+                <div class="dropdown-menu">
+                    {{ Twig.for("document", "orderDocuments", "translationMap[document.type] is defined") }}
+                        <a class="dropdown-item"
+                            href="/order-document/preview/{{ Twig.print("document.id") }}"
+                            target="_blank">
+                            {{ Twig.print(Twig.call("trans", [Twig.var("translationMap[document.type]")])) }}
+                        </a>
+                    {{ Twig.endfor() }}
+                </div>
+            </div>
+        {{ Twig.elseif("orderDocuments | length == 1 and ('order-document' in enabledRoutes or 'all' in enabledRoutes)") }}
```

Comment:
`('order-document' in enabledRoutes or 'all' in enabledRoutes)` ist immer `true` (s.o.)
`orderDocuments | length == 1 ` ist ebenfalls immer `true`, da in Zeile 31 bereits auf `orderDocuments | length > 0` geprüft wird

Reply:
Hier habe ich ebenfalls die Bedingung für die Route entfernt.
Hier findest du die Änderung: https://github.com/plentymarkets/plugin-ceres/pull/1361/commits/7af76963209912d0e5acaf12d4ab4053508a39c7


> orderDocuments | length == 1 ist ebenfalls immer true, da in Zeile 31 bereits auf orderDocuments | length > 0 geprüft wird

In diesen Block soll nur gesprungen werden, wenn es nur 1 Dokument gibt. Das ist der Fall wo dann der Button eine andere Aufschrift bekommt, und direkt zur Datei verweist. (Das ist die Änderung die du als erstes geschrieben hast.)